app/konnect/account-management/index.md

```diff
@@ -37,3 +37,9 @@ control plane. You should never have to deal with a license
 directly.
 
 For any license questions, contact your sales representative.
+
+## Geographic region management
+
+When you create a {{site.konnect_saas}} account, you select a [geographic region](/konnect/geo) for your instance. Geos are distinct deployments of {{site.konnect_short_name}} with objects, such as services and consumers, that are geo-specific. Only authentication is shared between {{site.konnect_short_name}} geos.
+
+Only Enterprise plans can select more than one geo.
```

Comment:
This is no longer accurate with the GA release. All Konnect customers, regardless of tiers will be provisioned all Konnect geos out of the box.